content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -225,6 +225,18 @@ crictl ps
 CONTAINER       IMAGE                                 CREATED           STATE      NAME    ATTEMPT    POD ID
 89db4553e1eeb   docker.io/library/nginx@sha256:...    19 seconds ago    Running    web     1          34533c6729106
 ```
+Once you identify the right container, you can get the logs for that container with `crictl`:
+
+```shell
+# Run these commands on the node where the kubelet is running
```

Comment:
You are checking the containers not the `kubelet`, right? `kubelet` runs on all nodes.

```suggestion
# Run these commands on the node where the container is running
```